snow/consensus/snowball/nnary_snowflake.go

```diff
@@ -9,7 +9,22 @@ import (
 	"github.com/ava-labs/avalanchego/ids"
 )
 
-var _ NnarySnowflake = (*nnarySnowflake)(nil)
+var (
+	_ NnarySnow   = (*nnarySnowflake)(nil)
+	_ ConsensusFactory = (snowflakeFactory{})
```

Comment:
Q: should this be exported same as snowball even though it's not used?

Probably not for now